acceptance-tests/acceptance_tests_suite_test.go

```diff
@@ -0,0 +1,50 @@
+package acceptance_tests
+
+import (
+	"encoding/json"
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"testing"
+)
+
+func TestAcceptanceTests(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "AcceptanceTests Suite")
+}
+
+var _ = SynchronizedBeforeSuite(func() []byte {
+	// Load config once, and pass to other
+	// threads as JSON-encoded byte array
+	var err error
+	config, err = loadConfig()
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
Please add explanations for expectations and their failures via the `optionalDescription` field.

This comment applies to pretty much every `Expect()`.

```suggestion
	Expect(err).NotTo(HaveOccurred(), "loading common config")
```